src/components/RuleItem.vue

```diff
@@ -193,6 +194,18 @@ export default {
       }
       this.constraints.push(null);
     },
+    addRefinement() {
+      if (this.refinements == null) {
+        // action is just a string, but when adding refinements, action becomes an an array
+        let action = this.action;
+        Vue.delete(this.rule, 'action');
+        
+        Vue.set(this.rule, 'action', [{}]);
+        Vue.set(this.action[0], 'rdf:value', action);
```

Comment:
There is no possibility to add more than one action, right?
Does that mean that "hardcoding" action[**0**] is ok?